### PR TITLE
[WebInterface] init TV email alerts handling

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot/pull/2766

image

no auth account
image

no ext account
image

ext account: setup stepper
image

image
image

image
image

image

image
image

image
image

alernative: no email received after 2min: (it keeps listening in bg but warns the user that there might be a problem)
image

### CODE REVIEW

**Review by Herklos — APPROVED** [review]
nice feature :+1:

File: Services/Interfaces/web_interface/static/js/common/util.js [hunk 1]
```diff
@@ -462,6 +462,12 @@ function selectFirstTab(nestedNavBar=undefined){
 }
 
 function copyToClipBoard(name, value) {
+    if(!navigator.clipboard){
+        create_alert(
+            "error",
+            "Browser security is preventing copy. Please manually copy this value"
```
Comment (Herklos) [inline Services/Interfaces/web_interface/static/js/common/util.js:468]: 👍

File: Services/Services_bases/trading_view_service/trading_view.py [hunk 2]
```diff
@@ -105,7 +105,7 @@ def get_type(self) -> None:
         return services_constants.CONFIG_TRADING_VIEW
 
     def get_website_url(self):
-        return "https://www.tradingview.com/"
+        return "https://www.tradingview.com/?aff_id=27595"
```
Comment (Herklos) [inline Services/Services_bases/trading_view_service/trading_view.py:108]: 👍

**Review by Herklos — APPROVED** [review]

**Comment by GuillaumeDSM** [comment]
up

**Comment by GuillaumeDSM** [comment]
requires https://github.com/Drakkar-Software/OctoBot/pull/2767

**Review by Herklos — APPROVED** [review]

**Pull request merged**